docs/v0.4/getting-started/docker-for-windows.md

```diff
@@ -181,38 +188,105 @@ NAME     LATEST IMAGE
 square   index.docker.io/$DOCKER_ID/square@sha256:ac089ca183368aa831597f94a2dbb462a157ccf7bbe0f3868294e15a24308f68   square.js   <empty>   <empty>   Ready    1m13s
 ```
 
-## create a deployer
+> Due to differences between Windows and Linux file permissions, local builds are not supported on Windows. We suggest building from a git repository as a workaround.
+
+## create a Knative deployer
+
+The [Knative Runtime](../runtimes/knative.md) is only available on clusters with Istio and Knative installed. Knative deployers run riff workloads using Knative resources which provide auto-scaling (including scale-to-zero) based on HTTP request traffic, and routing.
+
+```powershell
+riff knative deployer create knative-square --function-ref square --tail
+```
+
+After the deployer is created, you can see the hostname by listing deployers.
+
+```powershell
+riff knative deployer list
+```
+```
+NAME             TYPE       REF      HOST                                 STATUS   AGE
+knative-square   function   square   knative-square.default.example.com   Ready    28s
+```
+
+### invoke the function
+
+Knative configures HTTP routes on the istio-ingressgateway. Requests are routed by hostname.
+
+Look up the nodePort for the ingressgateway; you should see a port value like `30086`.
+
+```powershell
+$INGRESS_PORT = kubectl get svc istio-ingressgateway --namespace istio-system `
+  --output 'jsonpath={.spec.ports[?(@.port==80)].nodePort}'
+
+$INGRESS_PORT
+```
+
+Invoke the function by POSTing to the ingressgateway on the nodePort, passing hostname and content-type as headers.
+
+```powershell
+curl http://localhost:$INGRESS_PORT/ `
```

Comment:
Are you sure this port is exposed on localhost?

Reply:
works on my machine... do you have reason to expect it might not work ?

Reply:
worked for me too